### DISCUSSION TITLE
StreamGeometry  to PathGeometry

### DISCUSSION THREAD

**Discussion by leonidcan** [AvaloniaUI/Avalonia#17563] [Q&A]
How can I convert StreamGeometry  to PathGeometry?

**Comment by maxkatz6** [comment]
New API `PathGeometry.CreateFromGeometry` can be considered to be added, if SkiaSharp allows such functionality.

But first question to ask here is: why?

**Comment by leonidcan** [comment]
There is an idea to make an moving animation of a control along a path. But maybe there are other ways to do that?
